pages/exercises/[lessonSlug].tsx

```diff
@@ -45,14 +80,113 @@ const Exercises: React.FC<QueryDataProps<GetAppQuery>> = ({ queryData }) => {
     { text: 'exercises', url: `/exercises/${currentLesson.slug}` }
   ]
 
+  const exercise = mockExercises[exerciseIndex]
+
   return (
     <Layout title={currentLesson.title}>
+      {exercise ? (
+        <Exercise
+          exercise={exercise}
+          setExerciseIndex={setExerciseIndex}
+          lessonTitle={currentLesson.title}
+          showPreviousButton={exerciseIndex > 0}
+          showSkipButton={exerciseIndex < mockExercises.length - 1}
+        />
+      ) : (
+        <ExerciseList
+          tabs={tabs}
+          setExerciseIndex={setExerciseIndex}
+          lessonTitle={currentLesson.title}
+        />
+      )}
+      {alerts && <AlertsDisplay alerts={alerts} />}
+    </Layout>
+  )
+}
+
+type ExerciseProps = {
+  exercise: ExerciseCardProps
+  setExerciseIndex: React.Dispatch<React.SetStateAction<number>>
+  lessonTitle: string
+  showPreviousButton: boolean
+  showSkipButton: boolean
+}
+
+const Exercise = ({
+  exercise,
+  setExerciseIndex,
+  lessonTitle,
+  showPreviousButton,
+  showSkipButton
+}: ExerciseProps) => {
+  return (
+    <div className="w-75 mx-auto">
+      <button
+        className="btn ps-0 d-flex align-items-center"
+        onClick={() => setExerciseIndex(-1)}
+      >
+        <ArrowLeftIcon size="medium" />
+        <div className="p-3">
+          <Text size="xs">Exit</Text>
```

Comment:
I'm wondering. This exists in the design but do we need the text `Exit`? Isn't the icon enough to indicate that it's an exit button?

Reply:
Yeah, we can remove it and label the exit button with `aria-label="Exit"`.

Reply:
Done.